plugins/statuspage/dataset_client.py

```diff
@@ -44,3 +44,30 @@ def update(self, component_id, status):
         """
         patch = {"component": {"status": status}}
         return self.client.update_component(component_id, patch)
+
+    def create_incident_investigation(
+        self, name, component_id, body=None, component_status="partial_outage"
+    ):
+        """Create a new incident given a list of affected component ids.
+
+        :param name:                The name of the dataset.
+        :component_id:              The component id of the dataset
+        :param body:                The initial message, created as the first incident update.
+        :param incident_status:     The status of the incident e.g. investigating, identified, resolved
```

Comment:
Out of date comment - the def was streamlined because we don't need to automate closing of incidents.